code/game/atoms_movable.dm

```diff
@@ -19,6 +19,7 @@
 	var/inertia_move_delay = 5
 	var/pass_flags = 0
 	var/moving_diagonally = 0 //0: not doing a diagonal move. 1 and 2: doing the first/second step of the diagonal move
+	var/list/mobs_in_contents // This contains all the client mobs within this container

```

Comment:
Some generic on_cointainer_move or similar might be tad better and would have some use outside this system.